### DISCUSSION TITLE
Question regarding 'groei' configuration

### DISCUSSION THREAD

**Discussion by BosJelle** [nicwortel/indexfondsenvergelijken.nl#35] [Q&A] [answered]
Hi! 

Hope this question finds you well. 

When you change the configuration 'groei' on the website indexfondsenvergelijken.nl, the brokers and their costs also change. 
For instance, when you set 'dividend' to 0 the cheapest broker with the highest marketkapitalisation is DeGiro with IWDA + IEMM. If you change the dividend to 2,5 the three biggest banks are the cheapest with the highest marketcapitalisation with NT World + NT EM + NT SC. This is probably because of the dividend 'lekkage' at DeGiro. 

My question: where is the configuration 'groei' based on? Because if you change it the selection also changes.

**Comment by nicwortel** [comment, marked as answer]
Hi! Sorry for the late reply, I seem to have missed this question.

Changing the growth and dividend values should not change the selection of funds/brokers, but can influence their relative costs and therefor the ordering of the results. That's because the return impacts the value of your portfolio over time, and the value of your portfolio impacts the costs.

Groei and dividend together form the return, which is based on the performance of the MSCI ACWI index since 1987. The annualised performance is currently 7.85 so the 7.5% is a little outdated, but not too far off.
The dividend of 2.5% is a rough estimation. The 5% groei is simply 7.5% total return minus 2.5% dividend.

Because the future return, growth and dividend are the biggest unknown factor in the whole simulation, I've made them configurable so that people can play with the values to test different scenarios.
